src/pages/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -144,6 +145,16 @@ Prior to running the `ampd` daemon, you will need to set up your wallet with dev
 
     Running `ampd` without any command will run the daemon. A state file will be created if it doesn't yet exist. Its default location is `~/.ampd/state.json`, which can be overridden by passing `--state [path]`
 
+## Check if your verifier is in an active verifier set
+
+Query the prover to get the active verifier set. From here, you can check to see if your verifier is in the set:
+
+```bash
+axelard q wasm contract-state smart $PROVER_AVALANCHE '"get_verifier_set"' --node $RPC
```

Comment:
Where do I get the prover? As a verifier I don't know this. Isn't there a prover for each chain I'm verifying for?